config/servicemappings/accesscontextmanager.yaml

```diff
@@ -356,3 +356,30 @@ spec:
                 group: resourcemanager.cnrm.cloud.google.com
               targetField: "number"
               valueTemplate: "projects/{{value}}"
+    - name: google_access_context_manager_service_perimeter_resource
+      kind: AccessContextManagerServicePerimeterResource
+      idTemplate: "{{perimeter_name}}/{{resource}}"
+      idTemplateCanBeUsedToMatchResourceName: false
+      resourceAvailableInAssetInventory: false
+      v1alpha1ToV1beta1: true
+      storageVersion: v1alpha1
+      autoGenerated: false
```

Comment:
We can drop this property. See bullet point 7 here: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md#add-the-resource-config-to-service-mappings